### DISCUSSION TITLE
Photoprism & Yunohost

### DISCUSSION THREAD

**Discussion by tootbrute** [photoprism/photoprism#699] [General]
Hello!

Great work so far on Photoprism so far!

I use Yunohost to self-host lots of services on my own server. [I have introduced Photoprism to people on the Yunohost forum](https://forum.yunohost.org/t/photoprism-as-google-photos-replacement-my-experience/13510) because I think many people will be looking for a photo database solution like this now that Google Photos will start charging for storage.

Hope many more users, developers, (and donations!) come your way soon with the Google Photos exodus.

My personal top 3 features wish list are in the UX dept: batch edit keywords, batch tag people, and batch delete photos.

My experience geting it to run on my Yunohost server [here](https://blog.arkadi.one/photoprism-a-google-photos-replacement).

Cheers!

**Comment by graciousgrey** [comment]
Thanks a lot for your feedback and for writing a guide to help other Yunohost users with the setup!

Maybe we should add a section in our docs with guides for setups using different tools/infrastructures.

Regarding your feature requests: Batch edit, Facial recognition and Deletion of files using the interface will be the first things we implement once we have a stable release (see: https://github.com/photoprism/photoprism/projects/5) :)
In your blog post you mention "poor support for old video formats". We worked on video formats the last week, so feel free to test those again and let us know if there are formats that still do not work.

Have a nice weekend!

**Reply by tootbrute** [reply]
Awesome! You too.

Just sponsored this project. Keep up the good work!

Reindexing all my originals to check it out.
